Source/WebCore/css/makeprop.pl

```diff
@@ -518,7 +518,54 @@ sub sortWithPrefixedPropertiesLast
 CSSPropertyID findCSSProperty(const char* characters, unsigned length)
 {
     auto* value = CSSPropertyNamesHash::in_word_set(characters, length);
-    return value ? static_cast<CSSPropertyID>(value->id) : CSSPropertyInvalid;
+    return value ? static_cast<CSSPropertyID>(value->id) : CSSPropertyID::CSSPropertyInvalid;
+}
+
+ASCIILiteral nameLiteral(CSSPropertyID id)
```

Comment:
My least favorite part of this is calling everything an "id". I wish the type name and the argument names didn’t use that terminology.